src/providers/AuthProvider.tsx

```diff
@@ -0,0 +1,56 @@
+import { OAuth2Token } from "@badgateway/oauth2-client";
+import React from "react";
+import { useContext, useState } from "react";
+
+interface IAuthProvider {
+  jiraToken?: OAuth2Token;
+  githubToken?: OAuth2Token;
+  trelloToken?: OAuth2Token;
+  setJira: (newToken: OAuth2Token) => void;
+  setGithub: (newToken: OAuth2Token) => void;
+  setTrello: (newToken: OAuth2Token) => void;
```

Comment:
Trello doesn't support OAuth 2.0.

Reply:
Great, I guess we go for OAuth 1.0. This might make some parts of our work unusable for you guys.